.github/workflows/ci.yaml

```
@@ -67,6 +67,14 @@ jobs:
         if: always()
         run: postgrest-with-postgresql-9.5 postgrest-test-spec
 
+      - name: Run query cost tests against all PostgreSQL versions
+        if: always()
+        run: postgrest-with-all postgrest-test-spec-querycost
+
+      - name: Run doctests
+        if: always()
+        run: nix-shell --run postgrest-test-doctests
```

```suggestion
        run: postgrest-test-doctests
```

Any reason why not like this?

---

I couldn't get it work properly outside nix-shell yet... It fails because ghci somehow picks up the wrong GHC packages. It works for me locally by setting GHC_PACKAGE_PATH explicitly, but then that fails in CI... need to investigate